### PR TITLE
add content disposition header when request raw object

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**
Also this functionality was extracted to external class

### CODE REVIEW

File: src/main/java/org/ow2/proactive/catalog/util/RawObjectResponseCreator.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        String name = rawObject.getName();
+
+        byte[] bytes = rawObject.getRawObject();
+
+        ResponseEntity.BodyBuilder responseBodyBuilder = ResponseEntity.ok().contentLength(bytes.length);
+
+        try {
+            String contentDispositionFileName = name;
+
+            //add the .xml extension to contentDispositionFileName for workflow if the extension was not yet in name
```
Comment (MaelAudren) [inline src/main/java/org/ow2/proactive/catalog/util/RawObjectResponseCreator.java]: rephrase in "for the workflow if the extension was not in the variable 'name'"

File: src/test/java/org/ow2/proactive/catalog/util/RawObjectResponseCreatorTest.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                                                          "name",
+                                                          "object",
+                                                          "testContentType",
+                                                          1400343L,
+                                                          "commit message",
+                                                          Collections.emptyList(),
+                                                          new byte[0]);
+        ResponseEntity responseEntity = rawObjectResponseCreator.createRawObjectResponse(rawObject);
+        assertThat(responseEntity).isNotNull();
+        assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
```
Comment (MaelAudren) [inline src/test/java/org/ow2/proactive/catalog/util/RawObjectResponseCreatorTest.java]: The wrong content type return an OK response ?

Comment (ShatalovYaroslav) [reply src/test/java/org/ow2/proactive/catalog/util/RawObjectResponseCreatorTest.java]: exactly. This what we decided long time ago in this issue:
https://github.com/ow2-proactive/catalog/issues/41

File: src/main/java/org/ow2/proactive/catalog/util/RawObjectResponseCreator.java (hunk 1 above)
Comment (ShatalovYaroslav) [reply src/main/java/org/ow2/proactive/catalog/util/RawObjectResponseCreator.java]: ok

**Pull request merged**